src/components_redux/Radios.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+
+export default class Radios extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      type: " "
```

Comment:
you could use `text` as default value, so something is selected at the beginning.

Reply:
Okay, I will put the default value.